docs/blog/newsletter-april-2020.rst

```diff
@@ -0,0 +1,98 @@
+* Some people thought that salaries in expensive cities seemed low; others seemed to realize just how low their own salaries were compared with the average for their locale.
+* Some felt that the data gave them what they needed to negotiate raises or, more generally, better compensation packages! Just what we like to hear.
+
+The education question in particular produced a plethora of responses. Some were surprised at how many respondents had college degrees; many documentarians shared stories about their degrees in unrelated fields. Not many of the community have degrees or formal training in technical writing/communication (but some do!), and a fair few are interested in pursuing more training. Education -- whether you have a degree, or what your degree is in -- seems to matter less the more experience you have.
+
+General agreement that compulsive curiosity and desire to learn are key attributes of a successful documentarian.
+
+------------------------------------------------
+Book club: an intro to  'Every Page is Page One'
+------------------------------------------------
+
+This month members of our virtual book club discussed the first few chapters of Mark Baker's ‘Every Page is Page One’ (EPPO). For those who haven't read it, we thought we'd give a summary of the ideas.
+
+The core premise of EPPO is that users rely heavily on web search for help with when they have problems with software. This means you can't rely on them navigating nicely through your doc site structure, starting at your helpful introductory page - any page of your docs can be your user's page one.
+
+There's no disagreement from our book club members on the importance of search engines like Google as an entry point to docs. Not surprisingly, docs locked behind paywalls and/or logins can negatively affect user satisfaction. The community suggests, however, that these barriers can help combat competing content available on the web whether through Reddit, YouTube or other frequently trafficked learning platforms.
+
+Another observation from Baker is that, when  it comes to finding what they’re looking for on the web, users opt for the path of least resistance. He calls this "information foraging". So if you want to optimize for this foraging behaviour, that means you need to make sure your content is both ‘nutritious’ (information rich) and ‘easy to catch’.
+
+---------
+Job posts
+---------
+
+* `Technical Writer <https://jobs.writethedocs.org/job/193/technical-writer/>`__
+   Ably, London - full-time
+* `Technical Copywriter <https://jobs.writethedocs.org/job/194/technical-copywriter/>`__
+   Ably, London - full-time
+
+*To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_, *or check out the* `#job-post-only channel <https://app.slack.com/client/T0299N2DL/C09E989K5>`__ *on Slack.*
+
+--------------------------
+Community events coming up
+--------------------------
+
```

Comment:
Do we need an short intro on remoteness here? Or a pointer towards hosting remote meetups from Chris?

Reply:
I'll put something in. Do we have something to link to from Chris?

Reply:
Found it!